### ISSUE TITLE
[QUESTION] Different Keywords

### ISSUE THREAD

**Issue by DeRealMorgan** [Agnee-Transmissions/flutter-tf-speech#2]
Hey! I just installed your plugin and tried it out, works quite nicely!

I need keyword recognition in my app but I never touched TensorFlow. 
1. Is it possible to change the keywords so I can use my own?
2. As far as I know tf requires training data to improve the accuracy. My usecase needs to recognize these words no matter who is speaking. Is that possible?

Thanks in advance!
